front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -122,17 +157,15 @@ const StcdmResults = ({
             isReadOnly
             hideAttribution
             showStdcmAssets
+            isFeasible={!hasConflictResults}
             setMapCanvas={setMapCanvas}
-            pathGeometry={selectedSimulation.outputs?.pathProperties.geometry}
-            simulationPathSteps={selectedSimulation.outputs?.results.simulationPathSteps}
+            pathGeometry={outputs?.pathProperties?.geometry}
+            simulationPathSteps={simulationPathSteps}
           />
         </div>
       </div>
-      {isDebugMode && pathTrackRanges && selectedSimulation.outputs && (
-        <StdcmDebugResults
-          pathTrackRanges={pathTrackRanges}
-          simulationOutputs={selectedSimulation.outputs}
-        />
+      {isDebugMode && pathTrackRanges && hasSimulationResults && !hasConflictResults && (
```

Comment:
I think `hasSimulationResults` is enough, they be both true at the same time

```suggestion
      {isDebugMode && pathTrackRanges && hasSimulationResults && (
```